resources/quickstart/waas/web-quickstart.mdx

```diff
@@ -95,72 +89,51 @@ To retrieve these values for configuration within your application, follow these
 />
 </div>
 
-For more information on the Particle dashboard, take a look at [this video](https://www.youtube.com/watch?v=d7DYCMNDmjo&ab_channel=ParticleNetwork).
+For more information on the Particle dashboard, take a look at [this video](https://www.youtube.com/watch?v=d7DYCMNDmjo&ab_channel=ParticleNetwork) or visit the [dashboard quickstart](/resources/quickstart/dashboard).
 
-With your `projectId`, `clientKey`, and `appId` retrieved, you'll need to configure Particle Auth within your application according to the SDK you're using. For Particle Auth, a new instance of `ParticleNetwork` can be created for central control over the whole SDK, including facilitating social logins. To initialize `ParticleNetwork`, use a snippet similar to the example below:
+With your `projectId`, `clientKey`, and `appId` retrieved, you'll need to configure Particle Auth within your application through `AuthCoreContextProvider`, the primary React component responsible for initializing Particle Auth.
 
-```javascript JavaScript
-import { ParticleNetwork } from '@particle-network/auth';
-
-const particle = new ParticleNetwork({
-    projectId: process.env.REACT_APP_PROJECT_ID,
-    clientKey: process.env.REACT_APP_CLIENT_KEY,
-    appId: process.env.REACT_APP_APP_ID,
-    chainName: 'Ethereum',
-    chainId: 5, // Goerli
-    wallet: { displayWalletEntry: true },
-  });
-```
+Within your `src/app/layout.tsx` file, or in an equivalent location, wrap `{children}` (your application component) with `AuthCoreContextProvider`.
 
-Or, more likely, if you're using the new Auth Core SDK and configuring your application through `AuthCoreContextProvider` (imported from `@particle-network/auth-core-modal`), which wraps your primary React component within the `index` file, you can follow a structure such as the one below:
+Below is an example of how `AuthCoreContextProvider` can be configured.
 
-```javascript JavaScript
-import { AuthCoreContextProvider } from '@particle-network/auth-core-modal';
-
-ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
-  <React.StrictMode>
-    <AuthCoreContextProvider
-      options={{
-        projectId: process.env.REACT_APP_PROJECT_ID,
-        clientKey: process.env.REACT_APP_CLIENT_KEY,
-        appId: process.env.REACT_APP_APP_ID,
-        wallet: {
-          visible: true,
-        },
-      }}
-    >
-      <App />
-    </AuthCoreContextProvider>
-  </React.StrictMode>
-)
-```
+import { InteractiveLayout } from '/snippets/InteractiveLayout.mdx';
```

Comment:
This needs to be replicated for Mobile WaaS quickstart, Desktop AA quickstart, Mobile AA quickstart, and the customization pages (these can wait until new Particle Connect SDK is ready).

Not a huge priority but definitely the goal to have consistent interactive config snippets within these more granular quickstarts

Reply:
yep, this is definitely the goal.